cockatrice/src/window_main.cpp

```diff
@@ -415,8 +415,8 @@ void MainWindow::loginError(Response::ResponseCode r,
         case Response::RespContextError:
             QMessageBox::critical(this, tr("Error"),
                                   tr("An internal error has occurred, please try closing and reopening your client and "
-                                     "try again. If the error persists try updating your client to the most recent "
-                                     "build and if need be contact your software provider."));
+                                     "trying again. If the error persists try updating your client to the most recent "
+                                     "build and if needed contact software's developers."));
```

Comment:
That sentence sounds still a little weird to me.
Can we rephrase and simplify that maybe?

Reply:
Maybe:
An internal error has occurred, please close and reopen Cockatrice before trying again. If the error persists, ensure you are running the latest version of the software and if needed contact the software developers.